src/com/owncloud/android/operations/RefreshFolderOperation.java

```diff
@@ -349,11 +349,6 @@ private void synchronizeData(ArrayList<Object> folderAndFiles, OwnCloudClient cl
         // get 'fresh data' from the database
         mLocalFolder = mStorageManager.getFileByPath(mLocalFolder.getRemotePath());
 
-        // parse data from remote folder 
-        OCFile remoteFolder = FileStorageUtils.fillOCFile((RemoteFile) folderAndFiles.get(0));
```

Comment:
> If you miss a local property, such as availableOfflineStatus (favorite before), it could just be copied in the same way.

Even better, apply the change @jabarros made later, modifying ```saveFolder``` so that it doesn't change ```availableOfflineStatus``` at all. We should change at some moment ```FileDataStorageManager.saveFolder``` and ```saveFile``` so that only properties that might really have changed are updated.